lib/sentry/plug_context.ex

```diff
@@ -123,24 +123,46 @@ defmodule Sentry.PlugContext do
       cookies: handle_data(conn, cookie_scrubber),
       headers: handle_data(conn, header_scrubber),
       env: %{
-        "REMOTE_ADDR" => remote_address(conn.remote_ip),
-        "REMOTE_PORT" => Plug.Conn.get_peer_data(conn).port,
+        "REMOTE_ADDR" => remote_address(conn),
+        "REMOTE_PORT" => remote_port(conn),
         "SERVER_NAME" => conn.host,
         "SERVER_PORT" => conn.port,
         "REQUEST_ID" => Plug.Conn.get_resp_header(conn, request_id) |> List.first()
       }
     }
   end
 
-  defp remote_address(address) do
-    address
-    |> :inet.ntoa()
-    |> case do
-      {:error, _} ->
-        ""
+  defp remote_address(conn) do
+    if header_value = get_header(conn, "x-forwarded-for") do
```

Comment:
Should the module documentation mention this behaviour for people that have other headers (for this purpose) to know which header to use?

Reply:
I found`X-Originating-IP`, `X-Remote-IP`, `X-Remote-Addr` for the same usage. However I don't known if some proxies still use these headers and the expected behavior when you have several different headers.